port/unix/omrcpu.c

```diff
@@ -199,7 +205,18 @@ omrcpu_get_cache_line_size(struct OMRPortLibrary *portLibrary, int32_t *lineSize
 	} else {
 		rc = OMRPORT_ERROR_INVALID_ARGUMENTS;
 	}
-#endif /* defined(RS6000) || defined (LINUXPPC) || defined (PPC) */
+#elif defined (OSX)
+	int queryPath[] = {CTL_HW, HW_CACHELINE};
+	int64_t result = 0;
+	size_t resultSize = sizeof(result);
+	int status = sysctl(queryPath, 2, &result, &resultSize, NULL, 0);
+	if (0 == status) {
+		*lineSize = (int32_t) result;
```

Comment:
This also needs the same code as above:
```c
 	if (NULL != lineSize) {
 		*lineSize = PPG_mem_ppcCacheLineSize;
 		rc = 0;
 	} else {
 		rc = OMRPORT_ERROR_INVALID_ARGUMENTS;
 	}
```
to handle the case where lineSize is null

Reply:
Good idea.  Error checking cleaned up and test case re-run.